src/tvheadend/Settings.h

```diff
@@ -31,6 +31,22 @@ namespace tvheadend {
   class Settings {
   public:
 
+    // Default values.
+    static const std::string DEFAULT_HOST;
+    static const int         DEFAULT_HTTP_PORT           = 9981;
```

Comment:
IMO it would make sense to define all these values in the newly introduced `.cpp` file, currently if you're looking at just the `.cpp` file you may come under the impression that those three constants are the only ones available in the class. Up to you though.